pytest/functional/hs_discover/test_discover_api.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import json
+
+
+@pytest.mark.django_db
+def test_search_term_found_in_title(admin_client, public_resource_with_metadata):
```

Comment:
I really like the testing patterns your defining with pytest.

Reply:
I'd like to see a test around an empty result set

Reply:
I'd also like to see a test around the 520

Reply:
It's satisfying to incorporate my older 2019 stub work that was deployed dark to enable developers to take advantage of this powerful approach. You and I are aware of the benefits and as others onboard will be good.

Reply:
As far as the 520 that exists for the unknown unknowns. All of the known possible error combinations have been given codes and helpful messages. Commit bca65401f